builds/fabric8-che/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/rh-che.properties

```diff
@@ -2,3 +2,6 @@
 # Endpoints for obtiaining Github / OpenShift Online tokens based on Keycloak token
 che.keycloak.oso.endpoint=NULL
 che.keycloak.github.endpoint=NULL
+
+# Register GitHub access token without client id and secret
+oauth.github.forceactivation=false
```

Comment:
maybe should be true by default in our case (rh-che)? so we don't need to customize it in our template